### PR TITLE
feat: Get Installed and System Apps Info

### PR DESCRIPTION
**Pull request by shashank2086**
Getting all Installed and System apps of your device with Name and Main Activity for easily selecting and starting automation of any Application.

### CODE REVIEW

File: lib/commands/apps.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ *  Gets system and installed applications info
+ *  The information consists of :-
+ *   - packageName
+ *   - appName
+ *   - packageActivty
+ * */
+
+commands.mobileGetAppsInfo = async function () {
+    return await this.jwproxy.command('/appium/device/apps', 'GET', {});
```
Comment (mykola-mokhnach) [inline lib/commands/apps.js:12]: please test your PRs before publishing them. This call won't work as jwproxy only supports in-session endpoints and the `session/<id>` prefix will be automatically added to it

Comment (shashank2086) [reply lib/commands/apps.js:12]: Yes got it

**Comment by shashank2086** [comment]
Why functional test failed?

**Comment by mykola-mokhnach** [comment]
> Why functional test failed?

It seems to be a CI issue, retry could help

@shashank2086 Do you have time to finish this PR?

**Comment by shashank2086** [comment]
> 

For this CI issue right?

**Comment by mykola-mokhnach** [comment]
> For this CI issue right?

No, I mean to finish the work on this PR. 
What needs to be done:
- The minimum version of appium-uiautomator2-server package should be bumped
- Code formatitng
- Documentation (in code and in readme)

Check, for example, https://github.com/appium/appium-uiautomator2-driver/pull/530

**Comment by shashank2086** [comment]
> > For this CI issue right?
> 
> No, I mean to finish the work on this PR. What needs to be done:
> 
> * The minimum version of appium-uiautomator2-server package should be bumped
> * Code formatitng
> * Documentation (in code and in readme)
> 
> Check, for example, #530

Okay got it

File: lib/commands/apps.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ *  Gets system and installed applications info
+ *  The information consists of :-
+ *   - packageName
+ *   - appName
+ *   - packageActivty
+ * */
+
+commands.mobileGetAppsInfo = async function () {
+    return await this.uiautomator2.jwproxy.command('/appium/device/apps', 'GET', {});
+  };
```
[the remark this answers is not in the document]
Comment (shashank2086) [reply lib/commands/apps.js:13]: sorry which spaces?

[the remark this answers is not in the document]
Comment (KazuCocoa) [reply lib/commands/apps.js:13]: Made a suggestion about this

**Comment by KazuCocoa** [comment]
> The minimum version of appium-uiautomator2-server package should be bumped

is remaining. https://github.com/appium/appium-uiautomator2-driver/blob/master/package.json#L63

File: README.md [hunk 3]
```diff
@@ -418,6 +418,20 @@ Retrieves the information about the device under test, like the device model, se
 
 The extension returns a dictionary whose entries are the device properties. Check https://github.com/appium/appium-uiautomator2-server/blob/master/app/src/main/java/io/appium/uiautomator2/handler/GetDeviceInfo.java to get the full list of returned keys and their corresponding values.
 
+### mobile: getPackages
+
+Retrieves the information about the installed packages with its respective launchable activity.
```
Comment (mykola-mokhnach) [inline README.md:423]: we should also mention that package without launchable activities are not added to the list

File: lib/commands/apps.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+let extensions = {}, commands = {};
+
+/**
+ *  Gets system and installed applications info
```
Comment (mykola-mokhnach) [inline lib/commands/apps.js:4]: Please format the docstring properly according to jsdoc documentation rules. For example the `@return` section must be present. Check docs for other extansions for examples

File: README.md (hunk 3 above)
Comment (shashank2086) [reply README.md]: okay

File: lib/commands/apps.js (hunk 4 above)
Comment (shashank2086) [reply lib/commands/apps.js:4]: okay

**Pull request closed without merging**